### ISSUE TITLE
Compile for win32_x64 ( found 4 high severity vulnerabilities)

### ISSUE THREAD

**Issue by Berdmanfolk** [SAP/node-rfc#230]
Hello, @bsrdjan 
I try to compile from source, but again to get the same error: 
```
C:\node-rfc-2.4.0>npm install

> node-rfc@2.4.0 preinstall C:\node-rfc-2.4.0
> npm install cmake-js prebuild-install prebuild node-addon-api

npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.1 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

+ prebuild-install@6.0.0
+ node-addon-api@3.1.0
+ cmake-js@6.1.0
+ prebuild@10.0.1
added 1104 packages from 559 contributors and audited 1105 packages in 41.681s

48 packages are looking for funding
  run `npm fund` for details

found 3 high severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details

> node-rfc@2.4.0 install C:\node-rfc-2.4.0
> prebuild-install --tag-prefix -r napi || cmake-js rebuild

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.1 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

audited 1105 packages in 50.413s

48 packages are looking for funding
  run `npm fund` for details

found 3 high severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details

C:\node-rfc-2.4.0>node -v
v14.15.4

C:\node-rfc-2.4.0>npm audit

                       === npm audit security report ===


                                 Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance


  High            Denial of Service

  Package         https-proxy-agent

  Patched in      >=2.2.0

  Dependency of   typings [dev]

  Path            typings > typings-core > popsicle-proxy-agent >
                  https-proxy-agent

  More info       https://npmjs.com/advisories/593


  High            Machine-In-The-Middle

  Package         https-proxy-agent

  Patched in      >=2.2.3

  Dependency of   typings [dev]

  Path            typings > typings-core > popsicle-proxy-agent >
                  https-proxy-agent

  More info       https://npmjs.com/advisories/1184


  High            Denial of Service

  Package         http-proxy-agent

  Patched in      >=2.1.0

  Dependency of   typings [dev]

  Path            typings > typings-core > popsicle-proxy-agent >
                  http-proxy-agent

  More info       https://npmjs.com/advisories/607

found 3 high severity vulnerabilities in 1113 scanned packages
  3 vulnerabilities require manual review. See the full report for details.

C:\node-rfc-2.4.0>

```

**Comment by bsrdjan** [comment]
All vulnerabilities are related to development dependency `typings` and have no impact on run-time.

The `node-rfc` can't fix that but you can create the issue in `typings` repository.

**Comment by Berdmanfolk** [comment]
Sorry @bsrdjan 
I inserted old log, which was before.

Now these  vulnerabilities shows:
```
C:\node-rfc-main>npm install

> node-rfc@2.5.1 preinstall C:\node-rfc-main
> npm install cmake-js prebuild-install prebuild node-addon-api

npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

+ node-addon-api@4.0.0
+ prebuild-install@6.1.4
+ prebuild@10.0.1
+ cmake-js@6.2.1
updated 4 packages and audited 813 packages in 25.858s

46 packages are looking for funding
  run `npm fund` for details

found 4 high severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details

> node-rfc@2.5.1 install C:\node-rfc-main
> prebuild-install --tag-prefix -r napi || cmake-js rebuild

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

audited 813 packages in 31.025s

54 packages are looking for funding
  run `npm fund` for details

found 4 high severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details

C:\node-rfc-main>node -v
v14.15.4

C:\node-rfc-main>npm audit

                       === npm audit security report ===


                                 Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance


  High            Arbitrary File Creation/Overwrite due to insufficient
                  absolute path sanitization

  Package         tar

  Patched in      >=3.2.2 <4.0.0 || >=4.4.14  <5.0.0 || >=5.0.6 <6.0.0 ||
                  >=6.1.1

  Dependency of   prebuild [dev]

  Path            prebuild > node-ninja > tar

  More info       https://npmjs.com/advisories/1770


  High            Arbitrary File Creation/Overwrite due to insufficient
                  absolute path sanitization

  Package         tar

  Patched in      >=3.2.2 <4.0.0 || >=4.4.14  <5.0.0 || >=5.0.6 <6.0.0 ||
                  >=6.1.1

  Dependency of   prebuild [dev]

  Path            prebuild > nw-gyp > tar

  More info       https://npmjs.com/advisories/1770


  High            Arbitrary File Creation/Overwrite via insufficient symlink
                  protection due to directory cache poisoning

  Package         tar

  Patched in      >=3.2.3 <4.0.0 || >=4.4.15  <5.0.0 || >=5.0.7 <6.0.0 ||
                  >=6.1.2

  Dependency of   prebuild [dev]

  Path            prebuild > node-ninja > tar

  More info       https://npmjs.com/advisories/1771


  High            Arbitrary File Creation/Overwrite via insufficient symlink
                  protection due to directory cache poisoning

  Package         tar

  Patched in      >=3.2.3 <4.0.0 || >=4.4.15  <5.0.0 || >=5.0.7 <6.0.0 ||
                  >=6.1.2

  Dependency of   prebuild [dev]

  Path            prebuild > nw-gyp > tar

  More info       https://npmjs.com/advisories/1771

found 4 high severity vulnerabilities in 813 scanned packages
  4 vulnerabilities require manual review. See the full report for details.

C:\node-rfc-main>
```

**Comment by bsrdjan** [comment]
These four are from another external package, https://github.com/prebuild/prebuild, using vulnerable dependencies. I see no possibility to fix that in node-rfc.

Also in this case run-time is not affected.
